ceph/rbd/utils.py

```diff
@@ -199,3 +200,19 @@ def convert_size(size_bytes):
     p = math.pow(1024, i)
     s = round(size_bytes / p)
     return "%s%s" % (s, size_name[i])
+
+
+def get_disk_usage_for_image(node, image_spec):
+    """
+    Fetch disk usage of image using rbd du command
+
+    Args:
+        node: node on which command needs to be executed
+        pool: pool for which usage to be fetched
+        image: image for which usage to be fetched
+
+    Returns:
+        disk usage in json format
+    """
+    command = f"rbd du {image_spec} --format json"
```

Comment:
This is a CLI command execution and so it should be present in cli/rbd.py Please change the same. And you can add the json parsing of the output which is line 218 either in your test module directly or in a different method in workflows

Reply:
Done.